client/me/help/help-contact-form/index.jsx

```diff
@@ -28,12 +28,22 @@ import { selectSiteId } from 'state/help/actions';
 import { getHelpSelectedSite } from 'state/help/selectors';
 import wpcomLib from 'lib/wp';
 import HelpResults from 'me/help/help-results';
+import {
+	bumpStat,
+	recordTracksEvent,
+	composeAnalytics,
+} from 'state/analytics/actions';
 
 /**
  * Module variables
  */
 const wpcom = wpcomLib.undocumented();
 
+const trackSibylClick = ( event, helpLink ) => composeAnalytics(
+	bumpStat( 'sibyl_question_clicks', helpLink.id ),
+	recordTracksEvent( 'calypso_sibyl_question_click', {} )
```

Comment:
I was going to pass the question text or helplink id, but I'm not quite sure how tracks work. We're collecting number of clicks in MC stats, and I wasn't sure if tracks let us group things by the event name separately from the extra data.. if that's possible, I'd quite like to pass through some question data :)